bee-node/bee-node/src/entrynode/builder.rs

```diff
@@ -14,6 +14,7 @@ use bee_autopeering::{
 };
 use bee_gossip::Keypair;
 use bee_plugin_version_checker::VersionCheckerPlugin;
+use bee_rest_api::endpoints::InitConfigEntryNode;
```

Comment:
Rename to put the `Config` part last for consistency with other config objects

Reply:
Fixed